articles/install-guide/index.md

```diff
@@ -1,53 +1,207 @@
-> [!div class="button"]
-> [Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=quantum.quantum-devkit-vscode)
+   ```bash
+   conda install -c quantum-engineering iqsharp
+   ```
 
-Then, check our [quickstart guide](xref:microsoft.quantum.write-program?tabs=tabid-vscode) for step-by-step instructions for your first quantum program.
+1. Verify the installation
 
+    - Create a minimal Q# operation, by creating a file called `Operation.qs`, and and adding the following code to it:
 
-### Python ###
+    ```qsharp
+    namespace HelloWorld
+    {
+        open Microsoft.Quantum.Intrinsic;
+        open Microsoft.Quantum.Canon;
 
-If you are a Python user, check out our [getting started with Python and the Quantum Development Kit guide](xref:microsoft.quantum.install.python).
+        operation HelloQ() : Result
+        {
+            Message($"Hello from quantum world!");
+            return Zero;
+        }
+    }
+    ```
 
+    - Create a python program called `run_hello_q.py` to call the Q# `HelloQ()` operation:
 
-## Learn Quantum Computing with Q# ##
+      ```python
+      from HelloWorld import HelloQ
 
-* [Quantum Katas](https://github.com/Microsoft/QuantumKatas)
-* [Samples](https://github.com/Microsoft/Quantum)
-* [Quantum Computing Concepts](xref:microsoft.quantum.concepts.intro)
-* [Microsoft Quantum Blog](https://cloudblogs.microsoft.com/quantum/?ext)
-* [Q# Blog](https://devblogs.microsoft.com/qsharp/)
+      HelloQ.simulate()
+      ```
 
+    - Run the program:
 
+    ```bash
+    python run_hello_q.py
+    ```
+
+    - Verify the output. Your program should output the following lines:
+
+    ```bash
+    Hello from quantum world!
+    0
+    ```
+
+## Develop with Jupyter notebooks
+
+1. Pre-requisites
+
+    - [Python](https://www.python.org/downloads/) 3.6 or later
+    - [Jupyter notebooks](https://jupyter.readthedocs.io/en/latest/install.html)
+
+1. Install the `iqsharp` package:
+
+    ```bash
+    conda install -c quantum-engineering iqsharp
+    ```
+
+1. Verify the installation:
+
+    - Run the following command to start the notebook server:
+
+    ```bash
+    jupyter notebook
+    ```
+
+    - Browse to the URL shown on the command line. For example:
+
+    ```bash
+    http://localhost:8888
+    ```
+
+    - Create a jupyter notebook with a Q# kernel, and add the following code to the first notebook cell:
+
+    ```qsharp
+    operation HelloQ () : Unit {
+        Message($"Hello from quantum world!");
+    }
+    ```
+
+    - Run this cell of the notebook:
+
+    You should see `HelloQ` in the output of the cell. When running in jupyter notebooks, the Q# code is compiled, and the notebook outputs the name of the operation(s) that it finds.
+
+## Develop with C# on Windows, using Visual Studio
+
+1. Pre-requisites
+
+    - Visual Studio, version, with the .NET Core cross-platform development workload enabled
```

Comment:
Can/should we include a link here for them to download it?

Reply:
Since this is under the Visual Studio workflow, I think users need to install the .NET Core SDK through the Visual Studio Installer in this case?

Reply:
Adding a link to the Visual Studio download. Are there any restrictions on version?

Reply:
To your point on call to action, yes, definitely.

Reply:
@bettinaheim will know the exact restrictions on version, but if I recall, it's 16.3 or later.

Reply:
16.3 or later is for code completion, but I think we earlier versions just without the code completion feature.